README.md

```diff
@@ -1 +1,232 @@
+
+```
+bundle install
+```
+
+### Usage <a name="usage"></a>
+
+1. Once the setup is complete, ensure you are still in the directory containing the Recipe App files.
+
+2. To run the app, execute the following command:
+
+```
+rails server
+```
+
+3.If you have future features like GUI or interactive mode, follow the specific instructions provided for those features in the app's documentation.
+
+### Tests <a name="tests"></a>
+
+To run the tests ensure you are in the directory containing the test files.
+
+1. Run the tests using the following command:
+
+```
+bundle exec rspec
+```
+
+3. All tests should pass without any errors or failures, ensuring that the code and its methods are functioning correctly.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Luis Emilio Rojas Rojas**
+
+- [GitHub](https://github.com/lerfast)
+- [Twitter](https://twitter.com/lerfast)
+- [LinkedIn](https://www.linkedin.com/in/luis-emilio-rojas-rojas-6ba90a8a/)
+
+👤 **Micronaut Sebastián Hernandez**
+
+- GitHub: [@shm04](https://github.com/shm04)
+- LinkedIn: [Sebastian Hernandez](https://www.linkedin.com/in/sebastian-hernandez-munoz/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Features <a name="future-features"></a>
+
+- [x] **Add Views**
+- [x] **Add Forms**
+- [x] **Add tests**
+- [x] **Add validations**
+- [x] **Show Foods at index page**
+- [x] **Show information from database on every page**
+- [x] **Authentication User**
+- [x] **Show shopping list**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Dynamic list creating by the users using the interface**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+**I would like to thank Microverse for giving us the opportunity to learn and grow as developers and also I like to thank my family, they are all my support. 🌟**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+Here are the most common questions about this project.
+
+### `Can we use the application as an organization`
+
+### `Answer`
+Yes! Make sure you give us credit of the application. 
+
+## `Where can we reach out to you ?`
+
+### `Answer`
+You can reach out to the organisation team by email [luisemiliorojas@hotmail.com](luisemiliorojas@hotmail.com) 
```

Comment:
- [ ] Kindly ensure that you don't use link element to style the email address, I got 404 error when I attempt to access it. You can use any other tag to do that, the main gist is that there should be no broken link in your README file. ℹ️

Reply:
In my repo, that bottom works fine, however, I changed the code for that button:
![image](https://github.com/lerfast/recipe-app/assets/111823606/11af3ad8-2cdc-48aa-9228-fc0e5c103a4c)
![image](https://github.com/lerfast/recipe-app/assets/111823606/9f91c5e3-7aa6-4b91-a790-3ffbd7b57fcf)
![image](https://github.com/lerfast/recipe-app/assets/111823606/15ea2239-df90-4a73-a39a-a08cb7ac305e)
![image](https://github.com/lerfast/recipe-app/assets/111823606/0289cdb3-60a8-4ffb-b3e5-970f230c3300)




So I hope that this will work for you

Reply:
- Well resolved